store/src/java/com/zimbra/cs/pop3/Pop3Handler.java

```diff
@@ -124,9 +124,15 @@ boolean startConnection(InetAddress remoteAddr) throws IOException {
 
     public void setLoggingContext() {
         ZimbraLog.clearContext();
-        ZimbraLog.addAccountNameToContext(accountName);
-        ZimbraLog.addIpToContext(clientAddress);
-        ZimbraLog.addOrigIpToContext(origRemoteAddress);
+        if (null != accountName) {
```

Comment:
The addition of null checks intrigues me a little due to the lack of these checks in the previous version. Was there a specific motivation for the addition of these checks (just avoiding any overhead of making those calls when unnecessary?)? Should I care? Am I missing something?

Reply:
No.  You are right.  These null check are not necessary.  I found that the ZimbraLog.addToContext method 
(called by addAccountNameToContext() etc.) initializes the specified context field value when the null is passed, but for this code, the context are all cleared in the first place, so null check does not make difference.  I'll remove them.